privacy-contracts/src/main/solidity/DefaultFlexiblePrivacyGroupManagementContract.sol

```diff
@@ -13,9 +13,9 @@
  * SPDX-License-Identifier: Apache-2.0
```

Comment:
@macfarla these solidity files need to be reviewed

Reply:
@pinges these solidity contracts are only called from Besu, right?

Reply:
@macfarla As long as the methods of the contract do not change we are OK, as the address of the contract will still be the same.